src/cli.js

```diff
@@ -203,5 +203,5 @@ Promise.resolve().then(() => {
   if (errored) { process.exitCode = 2 }
 }).catch(err => {
   console.log(err.stack) // eslint-disable-line no-console
-  process.exit(err.code || 1)
+  process.exit(parseInt(err.code) || 1)
```

Comment:
How about checking if `typeof err.code === "number")? I think it might be more transparent than `parseInt`.